package.json

```diff
@@ -33,7 +33,7 @@
     "preuninstall": "node preuninstall.js",
     "postpack": "rm -rf node_modules",
     "prepare": "npm run tsc && npm run jasmine",
-    "test": "npm run prepare && npm run jasmine",
+    "test": "npm run prepare",
```

Comment:
What's the purpose of this change?

Reply:
prepare already runs tests too, no need to `npm run jasmine` twice